hack/dev-setup-local

```diff
@@ -16,13 +16,59 @@
 
 source $(dirname "${0}")/common
 
+function __expose_docker_api_server() {
+    local _docker_api_server_secure_port="6443"
+    local _docker_kube_apiserver_service="kube-apiserver-docker-desktop"
+    kubectl -n kube-system get services ${_docker_kube_apiserver_service} &>/dev/null || \
+      kubectl -n kube-system expose pod ${_docker_kube_apiserver_service} --type=NodePort --name=kube-apiserver-docker-desktop --port=${_docker_api_server_secure_port} &>/dev/null
+    local _exposed_apiserver_port=$(kubectl -n kube-system get service ${_docker_kube_apiserver_service} -o jsonpath="{.spec.ports[?(.port == ${_docker_api_server_secure_port})].nodePort}")
+    echo ${_exposed_apiserver_port}
+}
+
+function __generate_docker_client_cert_files() {
+  local _cert=${1}
+  local _key=${2}
+  local _docker_user=$(k8s_username)
+  kubectl config view --raw -o jsonpath="{.users[?(.name == \"${_docker_user}\")].user.client-certificate-data}" | base64 -d > ${_cert}
```

Comment:
Maybe `--flatten` flag should be used as well. This is useful when the certificates and keys are stored in separate files. `--flatten` will put the content of these files in the config output.
To be clear, I propose
`kubectl config view --raw -o ...`  to be 
`kubectl config view --raw --flatten -o ...`